118 core tests directory hashing (#119)
hypertest-cloud/hypertest: packages/hypertest-docs/docs/getting-started/configuration.md

  // Custom hypertest build manifest filename
  buildManifestFileName: 'hypertest.manifest.json',

>   // TODO After code review (warning | error | salience)

Reviewer: ```suggestion
  // TODO After code review (warning | error | silence)
```

>   // Specifies the follow-up actions to be taken if data drift are detected between the deployed manifest and the local test suite. Can be: silence, warning or error

Reviewer: ```suggestion
  // Specifies the action to take if drift is detected between the deployed manifest and the local test suite. Can be: 'silence', 'warning', or 'error'. Defaults to 'warning'.
```
